### ISSUE TITLE
Fix for Thunderfury PPM (across all 1.12.1 -> 1.x major versions)

### ISSUE THREAD

**Issue by towneh** [elysium-project/database#48]
Below is a proposed fix for bug https://elysium-project.org/bugtracker/issue/3649

`UPDATE item_template SET spellppmRate_1 = 7.8945 WHERE entry = 19019;`

A previous fix was applied in https://github.com/elysium-project/server/issues/41 

However this was later broken again, which looks like it was caused with differences between the itemization downgrade .sql files. e.g. PPM value 7 is applied in https://github.com/elysium-project/database/blob/a8ccfc46521f646bab5656c0ae493d6d43b95a32/sql/1.09/1.09_pve_items.sql but no mention of this is in https://github.com/elysium-project/database/blob/a8ccfc46521f646bab5656c0ae493d6d43b95a32/sql/1.10/1.10_pve_items.sql leaving it at the default value of 5.52 from the master 1.12 db.

As I believe we are using the 1.12.1 (or at least late vanilla) proc mechanics, i.e. they can't proc off the proc itself and other damage enchant effects like poisons, it is only correct that the PPM value should be set so that it reflects the fixed 25% proc chance state.

Additional resource material http://forum.nostalrius.org/viewtopic.php?f=24&t=703

**Comment by neale111** [comment]
This is totally false, thunderfury or any weapon proc is never a ppm and were never a ppm, it has no limitations unlike weapon enchants.
Even in retail today thunderfury has no ppm and has a 20% proc rate and has been unchanged ever since.
Although it used to be 30% at the first stages and it could proc on procs.
But no ppm anywhere in the history of wow.

**Comment by towneh** [comment]
What's false? The fact that Elysium code uses spellppmRate values with Thunderfury and this needs to be thrown out out of the database and hard coded in the server repo instead? Or that you're not happy with us referring to it as ppm because this is what the database field is called?

**Comment by neale111** [comment]
There are no PPMs for thunderfury. 
All thunderfury retail 
videos show no PPM  from 2005-2007 and 2007-2017. All retail information on thunderfury provides no PPM from 2005-2017.

Only weapon enchants have PPM which retail sources provide.

Private server player speculations are false opinions and false preferences. A Blizzlike server don't cater to fallacies.

Thunderfury must not have PPM.

**Comment by stesve** [comment]
> Only weapon enchants have PPM which retail sources provide.

Dude... Go check how the spellppmrate field is handled for weaponattacks...

**Comment by neale111** [comment]
Calculation is PPm but it has to eesult to 20%

**Comment by stesve** [comment]
> Calculation is PPm but it has to eesult to 20%

Dude... Go check how the spellppmrate field is handled for weaponattacks...

**Comment by towneh** [comment]
But this is exactly what both this issues thread and the bugtracker is trying to propose here, a change in the PPM db value so the resulting % of procs meets the 20% rate.

**Comment by stesve** [comment]
> But this is exactly what both this issues thread and the bugtracker is trying to propose here, a change in the PPM db value so the resulting % of procs meets the 20% rate.
Exactly. For weapon procs the ppm is converted to a flatchance and a roll is performed.
